### PR TITLE
refactor(naming): rename flavors into systems - INNO-927

### PR DESCRIPTION
**Pull request by yhuard**
Renames "flavors" into "systems", updates the URLs and remove the "docs" fractal. The docs are now embedded with every system (not ideal, I know 😢)

### CODE REVIEW

**Comment by HAL-Patch-INNO** [comment]
Deploy preview for *europa-component-library* ready!

Built with commit a434470b8ada6b0345eab1215da65e3b9319244e

https://deploy-preview-595--europa-component-library.netlify.com

File: packages/styleguides/docs/src/testing/visual.md [hunk 1]
```diff
@@ -58,7 +58,7 @@ describe('my-component', () => {
 
 Of course, replace `my-component` by the name of the actual component and make sure the URL you provide exists.
 
-**Under the hood, the URL will be prefixed by `http://localhost:3000/flavors/{flavor}/components/preview/`**.
+**Under the hood, the URL will be prefixed by `http://localhost:3000/systems/{flavor}/components/preview/`**.
```
Comment (kalinchernev) [inline packages/styleguides/docs/src/testing/visual.md]: `systems/{system}` maybe

File: packages/styleguides/docs/theme-overrides/partials/header.nunj [hunk 2]
```diff
@@ -84,8 +84,8 @@
             </a>
           </li>
           <li class="ecl-navigation-menu__item" data-role="item">
-            <a href="/docs/flavors" class="ecl-navigation-menu__link ">
-            <span>Europa family and ECL flavors</span>
+            <a href="/docs/systems" class="ecl-navigation-menu__link ">
```
Comment (kalinchernev) [inline packages/styleguides/docs/theme-overrides/partials/header.nunj]: Gives 404 this one https://deploy-preview-595--europa-component-library.netlify.com/docs/systems

File: packages/styleguides/docs/src/testing/visual.md (hunk 1 above)
Comment (yhuard) [reply packages/styleguides/docs/src/testing/visual.md]: good point, thanks!

**Pull request merged**